python/rikai/spark/sql/codegen/mlflow_logger.py

```diff
@@ -119,6 +120,12 @@ def log_model(
             CONF_MLFLOW_POST_PROCESSING: post_processing,
             CONF_MLFLOW_ARTIFACT_PATH: artifact_path,
         }
+        for k, v in list(tags.items()):
```

Comment:
the problem here is that things like schema, flavor, version, and artifact path are required.
Instead of removing them from the dict, can we just handle the pre/post processing separately as optional (i.e., just don't put them in if they're None or empty string, `not <str_var>` should work).

Reply:
improved, thanks!